pkg/interop/blockchain/blockchain.go

```diff
@@ -3,21 +3,24 @@ Package blockchain provides functions to access various blockchain data.
 */
 package blockchain
 
-import "github.com/nspcc-dev/neo-go/pkg/interop/contract"
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop"
+	"github.com/nspcc-dev/neo-go/pkg/interop/contract"
+)
 
 // Transaction represents a NEO transaction. It's similar to Transaction class
 // in Neo .net framework.
 type Transaction struct {
 	// Hash represents the hash (256 bit BE value in a 32 byte slice) of the
 	// given transaction (which also is its ID).
-	Hash []byte
+	Hash interop.Hash256
```

Comment:
We can also change `blockchain.GetTransaction(hash []byte)`, `blockchain.GetTransactionHeight(hash []byte)` and `blockchain.GetContract(scriptHash []byte)` signatures.

Reply:
Yeah, fixed.